app/models/repaymentHistory/PaymentHistoryEntry.scala

```diff
@@ -17,10 +17,23 @@
 package models.repaymentHistory
 
 import java.time.LocalDate
+import java.time.Month.APRIL
 
 case class PaymentHistoryEntry(date: LocalDate,
                                description: String,
                                amount: Option[BigDecimal],
                                transactionId: Option[String] = None,
                                linkUrl: String,
-                               visuallyHiddenText: String)
+                               visuallyHiddenText: String) {
+
+  def getTaxYearEndYear: Int = {
```

Comment:
I believe this is something already implemented as a part of DateServiceInterface or you can add new method within DateService to expose this  logic; essentially all operations with LocalDate we would expect to be localised within this DateService